javascript/modules/client/README.md

```diff
@@ -69,7 +69,7 @@ const creationParams: ICreateParams = {
     name: "ERC20VotingDAO_" + Math.floor(Math.random() * 9999) + 1,
     metadata: "<ipfs-http-uri>",
   },
-  votingConfig: {
+  : {
```

Comment:
Is that correct? There should be a key named `proposals`

Reply:
Yes this documentation is not reviewed yet, so it may contain errors